oak_sign/src/lib.rs

```diff
@@ -31,44 +31,59 @@ pub const PUBLIC_KEY_TAG: &str = "PUBLIC KEY";
 pub const SIGNATURE_TAG: &str = "SIGNATURE";
 pub const HASH_TAG: &str = "HASH";
 
-/// Convenience struct that encapsulates `Ed25519KeyPair`.
-/// Named `KeyBundle` since it uses the `ring::signature::KeyPair` trait.
-pub struct KeyBundle {
-    private_key: Vec<u8>,
-    key_pair: Ed25519KeyPair,
+/// Convenience struct that encapsulates `ring::signature::Ed25519KeyPair`.
+pub struct KeyPair {
+    /// PKCS#8 v2 encoded private key and public key.
+    /// https://tools.ietf.org/html/rfc5958
+    ///
+    /// The encoded version is kept because a `parsed` key pair doesn't contain a method for
+    /// extracting private key bytes.
+    /// https://briansmith.org/rustdoc/ring/signature/struct.Ed25519KeyPair.html
+    encoded_key_pair: Vec<u8>,
```

Comment:
Why not call this field pkcs_8 something? There is nothing else that can be done with it without knowing its format anyways, so this seems better than just a comment.